packages/components/inputs/rich-text-utils/src/html/html.tsx

```diff
@@ -1,7 +1,15 @@
-/* eslint-disable consistent-return */
-import Html from 'slate-html-serializer';
+import Html, { type HtmlOptions } from 'slate-html-serializer';
 import flatMap from 'lodash/flatMap';
 import { MARK_TAGS, BLOCK_TAGS } from '../tags';
+import type { ReactNode } from 'react';
+
+type TSerializeObject = {
```

Comment:
Instead of a verb, maybe it's better to use a noun for types.
Perhaps we could use here something like `TSerializableObject`.